### DISCUSSION TITLE
Pvp Performance tracker addition

### DISCUSSION THREAD

**Discussion by PortSariim** [runelite/runelite#15833] [Q&A] [answered]
Greetings everyone, registered an account on github just to post this.
My request is that inside the pvp performance tracker it stores data about opponents equipment and percentages of styles.
For example 44% ranged 36% magic 20% melee, equipped elder chaos robes tormented bracelet spiked manacles, specced with armadyl godsword.

Or something along those lines.
I'm not an expert myself but I figure it should take some extra data but not much as it are just some numbers and item ids.

Would be great if this could be added, thank you a lot in advance for considering.

**Comment by Enriath** [comment, marked as answer]
Issues and suggestions for hub plugins must be made on that plugin's repository; they are not maintained by us. The repo is usually accessible from the Support link on the plugin.
